README.md

```diff
@@ -1,45 +1,55 @@
 # AIP++
 
-The full documentation of our application can be found at 
-https://cutt.ly/Mn78T3f
-### Running the application locally using Docker
+> A Software Tool for Finding Related Work in Academia
```

Comment:
For now the scope of the tool (at least the parsers) is (computing) system's conferences and journals, as we use the venue mapper to identify conferences and journals to map them consistently to the same acronym.

Reply:
Is it possible that only around 2% (130.000 publications in AtLarge dump vs. 5.6M publications in DBLP) of all publications satisfy that criterion for DBLP? Shouldn't more publications be in these system's conferences/journals? If so, I think the Venue Mapper library might miss quite a lot of venues. I could also be mistaken.

Reply:
Yeah, it misses many venues currently (have a look at the issue list!). However, this is a credible way of making sure we tag articles coming from these venues. Another would be to rely on the acronym in the DBLP citation ID. They have they structure of something like /conf/ipdps/... where ipdps is the acronym of the venue. If you have suggestions here, do share!